cli/http_util.rs

```diff
@@ -32,18 +33,25 @@ use url::Url;
 
 /// Create new instance of async reqwest::Client. This client supports
 /// proxies and doesn't follow redirects.
-pub fn create_http_client() -> Client {
+pub fn create_http_client(ca_file: Option<String>) -> Client {
   let mut headers = HeaderMap::new();
   headers.insert(
     USER_AGENT,
     format!("Deno/{}", version::DENO).parse().unwrap(),
   );
-  Client::builder()
+  let mut builder = Client::builder()
     .redirect(Policy::none())
     .default_headers(headers)
-    .use_rustls_tls()
-    .build()
-    .unwrap()
+    .use_rustls_tls();
+
+  if let Some(ca_file) = ca_file {
+    let mut buf = Vec::new();
+    File::open(ca_file).unwrap().read_to_end(&mut buf).unwrap();
+    let cert = reqwest::Certificate::from_der(&buf).unwrap();
```

Comment:
It will panic if file there's a problem with file or file contents - this function should return `Result`

Reply:
Maybe this can reuse `load_certs()` from `cli/ops/tls.rs`?

Aside: this reads and parses the file for every fetch, doesn't it? It's probably better to parse it at startup and cache the result.

Reply:
That could work. I guess then we could accept PEM files, which seems to be the norm. I'm finishing up integration tests right now. I'll see about refactoring after I've got all that checked in.

Reply:
Never mind, There's a `from_pem` on `reqwest::Certificate` that I overlooked. I don't know how I missed that. 

I guess I don't see why I'd use the `load_certs` from `cli/ops/tls.rs`. That function is actually returning a Vec of `rustls::Certificate`, which I'd have to convert to `reqwest::Certificate`. Seems like more work.

Reply:
> Maybe this can reuse `load_certs()` from `cli/ops/tls.rs`?

IIRC that's not compatible with `reqwest` - it's used in `listenTLS` which uses `tokio-rusttls` under the hood.

> Aside: this reads and parses the file for every fetch, doesn't it? It's probably better to parse it at startup and cache the result.

Yeah, that's very good point